lib/fetchStreamReadable/polyfills/fetchPolyfill.js

```diff
@@ -23,12 +23,15 @@ function isNativeStreamFetchSupported() {
 
   const majorVersion = parseInt(version.split('.')[0], 10);
 
-  return (name === 'chrome' && majorVersion >= 42) ||
-    (name === 'firefox' && majorVersion >= 65);
+  return (name === 'chrome' && majorVersion >= 43) ||
+    (name === 'firefox' && majorVersion >= 65) ||
+    (name === 'edge' && majorVersion >= 14) ||
+    (name === 'safari' && majorVersion >= 10) ||
```

Comment:
<img width="1006" alt="Captura de Pantalla 2022-07-04 a las 12 50 46" src="https://user-images.githubusercontent.com/14887264/177140294-99273d97-e4d8-4597-86c1-964a19b50f07.png">
According to the documentation Safari supports it from 10.1